### DISCUSSION TITLE
Why does renovate not support multiple tokens?

### DISCUSSION THREAD

**Discussion by Yannik** [renovatebot/renovate#30034] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### Please tell us more about your question or problem

When I posted [this](https://github.com/renovatebot/renovate/discussions/30013) discussion, I thought our requirement of using multiple personal/group access tokens (instead of a bot user) due to our gitlab permissions being managed by ldap group sync would be rather niche. After reading https://docs.renovatebot.com/gitlab-bot-security/, it seems like it is a must-have, and should be quite a common requirement.

Is there a reason for not supporting the use of multiple access tokens with renovate? Or is this feature already planned but not implemented yet?

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment, marked as answer]
FYI I believe GitLab have hardened their security plenty since we wrote that and it probably needs updating.

The feature is not planned, because it would complicate things quite a lot, and the benefit is not worth the complexity.
